lib/inspec/resources/nginx_conf.rb

```diff
@@ -63,6 +63,11 @@ def read_content(path)
     def parse_nginx(path)
       return nil if inspec.os.windows?
       content = read_content(path)
+
+      # Don't attempt to parse file if it contains only comments or is empty
+      # https://regexper.com/#%2F%5E%5Cs*%23%7C%5E%24%2F
+      return {} if content.split("\n").reject { |l| l =~ /^\s*#|^$/ } == []
```

Comment:
Might try this, it's a bit more idiomatic:
```
content.lines.reject { |l| l =~ /^\s*#|^$/ }.empty?
```

Reply:
Will do! I like yours better anyway.